views/home.hbs

```diff
@@ -1,9 +1,9 @@
 
 <div class="select__language">
-  <a href="/en/places">
-  <div class="english">ENGLISH</div>
+  <a href="/en/content">
+    <div class="english">English</div>
```

Comment:
hey sorry, I still quite liked using the variable - I just meant getting rid of the `this.` part, but keeping it as variable you pass in the `res.render` as that way you don't have to change it in multiple places.

Reply:
Or if there's a reason you did this let me know

Reply:
It's not gonna change it will always be Arabic in Arabic letter and English in English letters. So I think it should be fine this way.

Reply:
👍